Dockerfile

```diff
@@ -33,14 +33,41 @@ COPY chainspecs /creditcoin-node/chainspecs
 COPY test /creditcoin-node/test
 RUN source ~/.cargo/env && cargo build --release
 
+
+FROM node:16.13.0-bullseye-slim AS cli-builder
```

Comment:
Do we need to build the image using a non-root user? It creates a bunch of issues with the node base image and I don't think it is necessary

Checking hadolinter, for example, the last user which is used for running the image is the one that should have the least permissions but does not check users during build

Reply:
Makes sense to avoid using root when possible and this is the case. For me it conflicts with using the image provided by Node, which should be "the right tool". But I like your take on just don't using it if it conflicts with our practices and having the same OS and versions on both stages.